common.php

```diff
@@ -6610,6 +6651,9 @@ function ewww_image_optimizer_attachment_check_variant_level( $id, $type, $meta
 	if ( 'application/pdf' === $type && 10 !== (int) ewww_image_optimizer_get_option( 'ewww_image_optimizer_pdf_level' ) ) {
 		return $meta;
 	}
+	if ( 'image/svg+xml' === $type && ewww_image_optimizer_get_option( 'ewww_image_optimizer_svg_level' ) > 20 ) {
```

Comment:
If you don't plan doing cloud optimization for SVG, I think you should at least support backup of it - this is the added functionality your cloud provides to the user.
So I'll choose option 2, because svgcleaner might be too invasive even in this level.